packages/sdk/src/light-node/index.ts

```diff
@@ -41,6 +16,10 @@ export async function createNode(
 export async function createLightNode(
   options: CreateWakuNodeOptions = {}
 ): Promise<LightNode> {
+  if (!options.shardInfo || !options.contentTopics) {
+    throw new Error("Shard info must be set");
```

Comment:
What if I am running on non-TWN network/pubsub? Does it mean that I need to provide contentTopics on the node creation time?

What I do in Qaku is I accept `lightNode` as a parameter for waku-dispatcher and only re-create the dispatcher instance (which needs to know the `contentTopic` to create encoder/decoder), but keep the node to speed up switching between Q&As

The need to provide contentTopic on node creation completely breaks this and I'd have to either provide some dummy `contentTopic` or re-create node on each Q&A switch, which would slow down the loading.

I understand this is necessary to be able to discover the right nodes supporting the right shards, but not sure if it will be practical for the apps.

Reply:
@vpavlin are all the content topics for qwaku known beforehand or are they created dynamically?

Reply:
> What if I am running on non-TWN network/pubsub

js-waku is moving towards dropping custom pubsub topics
if non-TWN can be described with `ShardInfo` - then it can passed just as is and should work, otherwise it won't be supported 

note that not in this PR - I returned ability to run node with array of `pubsubTopics` but it will be removed next release

let's discuss if you think that we should handle it differently 

--- 

as @adklempner mentioned you can pass different content topics that will resolve to the same shard - this can be ensured as per spec